### PR TITLE
Create flowfuse-1-11-release.md

### PR DESCRIPTION
**Pull request by iskerrett**
Release 1.11 announcement post

### CODE REVIEW

File: src/blog/2023/08/flowfuse-1-11-release.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+<!--more-->
+## New FlowFuse Cloud Starter Tier  [#2328](https://github.com/flowforge/flowforge/issues/2328)
+
+It is now easier for Node-RED developers to get started with FlowFuse and Node-RED. The new starter tier allows developers to use two Node-RED instances and two remote device deployments. Ideal for creating proof of concepts or running a home automation system with Node-RED. 
+
+FlowFuse provides a cloud hosted version of Node-RED so developers don't need to worry about Node-RED installation or operation. This makes it a lot easier to get started with Node-RED and easier to maintain a running instance. 
+
+
+## FlowFuse API access now possible via personal access tokens [#14](https://github.com/flowforge/flowforge/issues/14)
+FlowFuse APIs are now acceessible via personal access tokens (PAT). This makes it possible to create automation scripts that interact with the FlowFuse platform using the API and authenticate the scripts with the PAT.
```
Comment (MarianRaphael) [inline src/blog/2023/08/flowfuse-1-11-release.md:25]:
```suggestion
FlowFuse APIs are now accessible via personal access tokens (PAT). This makes it possible to create automation scripts that interact with the FlowFuse platform using the API and authenticate the scripts with the PAT. 
```

File: src/blog/2023/08/flowfuse-1-11-release.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+## FlowFuse API access now possible via personal access tokens [#14](https://github.com/flowforge/flowforge/issues/14)
+FlowFuse APIs are now acceessible via personal access tokens (PAT). This makes it possible to create automation scripts that interact with the FlowFuse platform using the API and authenticate the scripts with the PAT. 
+
+## Usability Improvements to Device Management [#2294](https://github.com/flowforge/flowforge/issues/2334)
+
+A number of usability improvements have been added to the FlowFuse device management solution to make it more flexible and intuitive to use. These improvements include being able to associate devices at the application level allowing for easier editing of Node-RED instances on edge devices.
+
+## FlowFuse Rebranding [#119](https://github.com/orgs/flowforge/projects/1?pane=issue&itemId=34719640)
+
+Earlier in August, [FlowForge announce](https://flowfuse.com/blog/2023/08/flowforge-is-now-flowfuse/) a change to our company and product name to FlowFuse. Work has begun to change the product branding to FlowFuse. Most of the UI has been rebranded and the remaining points will be changed in the next release.
```
Comment (MarianRaphael) [inline src/blog/2023/08/flowfuse-1-11-release.md:33]:
```suggestion
Earlier in August, [FlowForge announced](https://flowfuse.com/blog/2023/08/flowforge-is-now-flowfuse/) a change to our company and product name to FlowFuse. Work has begun to change the product branding to FlowFuse. The UI has been rebranded and the remaining points will be changed in the next release.
```

File: src/blog/2023/08/flowfuse-1-11-release.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+## Upgrading FlowFuse
+
+[FlowFuse Cloud](https://app.flowforge.com) is already running 1.11.
+
+If you installed a previous version of FlowFuse and want to upgrade, our documentation provides a
+guide for [upgrading your FlowFuse instance](/docs/upgrade/).
+
+## Getting help
+
+Please check FlowFuse's [documentation](/docs/) as the answers to many questions are covered there. Additionally you can go the the [community forum](https://community.flowfuse.com) if you have
```
Comment (MarianRaphael) [inline src/blog/2023/08/flowfuse-1-11-release.md:79]:
```suggestion
Please check FlowFuse's [documentation](/docs/) as the answers to many questions are covered there. Additionally you can go to the [community forum](https://community.flowfuse.com) if you have
```

**Review by MarianRaphael — CHANGES_REQUESTED** [review]
Only one comment: I think we should also mention the new navigation structure: https://github.com/flowforge/flowforge/issues/2474. The second vertical navigation bar has now become a horizontal one.

File: src/blog/2023/08/flowfuse-1-11-release.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
+---
+title: FlowFuse 1.11 makes it easier to get started with FlowFuse and Node-RED
+subtitle: The new FlowFuse 1.11 release includes a new starter tier for FlowFuse Cloud, Personal Access Tokens for API access and improvements to device management.
```
Comment (ZJvandeWeg) [inline src/blog/2023/08/flowfuse-1-11-release.md:3]:
```suggestion
subtitle: Our latest release includes a new starter tier for FlowFuse Cloud, Personal Access Tokens for API access and improvements to device management.
```

File: src/blog/2023/08/flowfuse-1-11-release.md [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+date: 2023-08-31 
+authors: ["ian-skerrett"]
+image: /blog/2023/08/images/release-1-11-graphic.png
+tags:
+    - posts
+    - flowfuse
+    - releases
+---
+
+FlowFuse 1.11 introduces a new starter tier for FlowFuse Cloud that makes it easier to get started with FlowFuse and Node-RED.
```
Comment (ZJvandeWeg) [inline src/blog/2023/08/flowfuse-1-11-release.md:14]: We introduce the Team tier too?

File: src/blog/2023/08/flowfuse-1-11-release.md [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+## FlowFuse API access now possible via personal access tokens [#14](https://github.com/flowforge/flowforge/issues/14)
+FlowFuse APIs are now accessible via personal access tokens (PAT). This makes it possible to create automation scripts that interact with the FlowFuse platform using the API and authenticate the scripts with the PAT. 
+
+## Usability Improvements to Device Management [#2294](https://github.com/flowforge/flowforge/issues/2334)
+
+A number of usability improvements have been added to the FlowFuse device management solution to make it more flexible and intuitive to use. These improvements include being able to associate devices at the application level allowing for easier editing of Node-RED instances on edge devices.
+
+## FlowFuse Rebranding [#119](https://github.com/orgs/flowforge/projects/1?pane=issue&itemId=34719640)
+
+Earlier in August, [FlowForge announced](https://flowfuse.com/blog/2023/08/flowforge-is-now-flowfuse/) a change to our company and product name to FlowFuse. Work has begun to change the product branding to FlowFuse. The UI has been rebranded and the remaining points will be changed in the next release.
```
Comment (ZJvandeWeg) [inline src/blog/2023/08/flowfuse-1-11-release.md:33]: The URL changes too today, I think it should be mentioned

File: src/blog/2023/08/flowfuse-1-11-release.md [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+## Bug Fixes
+
+- Device running old snapshot [#132](https://github.com/flowforge/flowforge-device-agent/issues/132)
+
+
+
+## Community Contributions
+
+Thanks to our community members for their contributions to this release.
+
```
Comment (ZJvandeWeg) [inline src/blog/2023/08/flowfuse-1-11-release.md:51]:
```suggestion
```

File: src/blog/2023/08/flowfuse-1-11-release.md (hunk 5 above)
Comment (iskerrett) [reply src/blog/2023/08/flowfuse-1-11-release.md:14]: Team tier is just a rename of Premium. Correct? If this is the case, I don't think it is worth mentioning.

File: src/blog/2023/08/flowfuse-1-11-release.md (hunk 6 above)
Comment (MarianRaphael) [reply src/blog/2023/08/flowfuse-1-11-release.md:33]: https://github.com/flowforge/admin/issues/208

It's a separate configuration step after the release. Old instances will retain their previous domain configurations. Newly created instances will be created under the new domain.

https://flowforgeworkspace.slack.com/archives/C032Q63FGG1/p1693477042121479

Both https://app.flowforge.com/ and https://app.flowfuse.com/ will work.

Comment (iskerrett) [reply src/blog/2023/08/flowfuse-1-11-release.md:33]: Can you help draft some words on this topic? It is not clear to me what an end use needs to do. Is this in our new documentation that we can link to?

Comment (MarianRaphael) [reply src/blog/2023/08/flowfuse-1-11-release.md:33]: There will be no domain change today, it will remain unchanged https://app.flowforge.com/

Comment (iskerrett) [reply src/blog/2023/08/flowfuse-1-11-release.md:33]: Closing conversation as resolved. We won't add URL change to the blog post.

**Pull request merged**